include/log.h

```diff
@@ -15,14 +15,17 @@ extern "C"
 {
 #endif
 
+#if ENABLE_DEBUG_LOGGING
+
 	char* getFormattedTime(void);
 
 #define LOG(fmt, ...) fprintf(stdout, "[LOG:%s::%s::%d] " fmt "\n", getFormattedTime(), __FILE__, __LINE__ __VA_OPT__(, ) __VA_ARGS__);
 #define ERROR(fmt, ...) fprintf(stderr, "[ERR:%s::%s::%d] " fmt "\n", getFormattedTime(), __FILE__, __LINE__ __VA_OPT__(, ) __VA_ARGS__);
 
-#if ENABLE_DEBUG_LOGGING
```

Comment:
I guess this was fine, could the issue be misaligned #if and #endif.

Reply:
Yes you are right, it is corrected in commit https://github.com/intel/trustauthority-client-for-c/pull/2/commits/680a72fdbf4d6c047e9571f6e7ed540556be36a4

Reply:
moving `#if ENABLE_DEBUG_LOGGING` before LOG and ERROR statements will make the release build to not print LOG and ERROR statements.

Reply:
fixed in https://github.com/intel/trustauthority-client-for-c/pull/2/commits/759098e570f48de14eb72eb67843733306c12d34

Reply:
LOG and ERROR statements should be removed from `#else` otherwise they will not be logged for release builds.

Reply:
fixed https://github.com/intel/trustauthority-client-for-c/pull/2/commits/ee390e9bf65ba140cf8dc8068336aa00b1731b66